data/hai ships.txt

```diff
@@ -273,7 +273,7 @@ ship "Shield Beetle"
 	explode "large explosion" 50
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description "The Shield Beetle is a versatile warship used by the Hai to guard their merchant convoys."
+	description "This ship was first built millenia ago near the end of a more violent time in Hai history, and since then it has been the only heavy warship at the disposal of the Hai. The Shield Beetle is a versatile warship used by the Hai to guard their merchant convoys, or by the Unfettered to attack those very same convoys."
```

Comment:
Suggestions:
`The Shield Beetle`
`ago, near`
`history. Since then it has served as the`

Not sure about the last sentence.

Reply:
>`The Shield Beetle`

If you're talking about replacing `This ship`, I was trying to avoid using `The <ship>` for every single description.

Reply:
Yeah I had been looking into a rephrase if the last sentence which changed that use of `the shield beetle` into `it`, but couldn't think of a suitable change.

Reply:
"The Shield Beetle is a " -> "Shield Beetles are " may be better.

Reply:
The original sounds like a better sales pitch. Your version sounds more like something from a white paper.